model/secret_test.go

```diff
@@ -25,10 +25,26 @@ func TestSecret(t *testing.T) {
 			secret := Secret{}
 			g.Assert(secret.Match("pull_request")).IsTrue()
 		})
-		g.It("should pass validation")
+		g.It("should pass validation", func() {
+			secret := Secret{}
+			secret.Name = "secretname"
+			secret.Value = "secretvalue"
+			err := secret.Validate()
+			g.Assert(err).Equal(nil)
+		})
 		g.Describe("should fail validation", func() {
-			g.It("when no image")
-			g.It("when no event")
```

Comment:
removing this because it looks like not required from here
https://github.com/drone/drone/blob/master/model/secret.go#L47